PyRDF/backend/AWS.py

```diff
@@ -92,20 +131,78 @@ def invoke_root_lambda(client, root_range, script):
                 'filelist': str(root_range.filelist),
                 'friend_info': encode_object(root_range.friend_info)
             })
-            return client.invoke(
-                FunctionName='root_lambda',
-                InvocationType='Event',
-                Payload=bytes(payload, encoding='utf8')
-            )
+
+            # Maybe here give info about number of invoked lambda for awsmonitor
+            #self.logger.info(f'New lambda - 31')
+
+            while trials:
```

Comment:
This could be more explicit by stating "trials != 0", as that's what ends this `while`